CHANGELOG.next.asciidoc

```diff
@@ -171,6 +171,7 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 - Add support for polling system UDP stats for UDP input metrics. {pull}34070[34070]
 - Add support for recognizing the log level in Elasticsearch JVM logs {pull}34159[34159]
 - Add metrics for TCP packet processing. {pull}34333[34333]
+- Add `take over` mode for `filestream` for simple migration from `log` inputs {pull}34292[34292]
```

Comment:
```suggestion
- Add beta `take over` mode for `filestream` for simple migration from `log` inputs {pull}34292[34292]
```

If we want to consider this beta functionality we should state it right in the changelog.